dependencies.yaml

```diff
@@ -559,7 +559,8 @@ dependencies:
           - matrix: {cuda: "12.*"}
             packages:
               - rmm-cu12==24.6.*
-              - pynvjitlink-cu12
+              - --extra-index-url=https://pypi.anaconda.org/rapidsai-wheels-nightly/simple
```

Comment:
Think we already add this. It is where the RAPIDS 24.06 packages are coming from as well

Let's try dropping and see if things still work. We can always revisit if not

```suggestion
```